module/VuFind/src/VuFind/OAuth2/Entity/AuthCodeEntity.php

```diff
@@ -54,8 +54,7 @@ class AuthCodeEntity implements AuthCodeEntityInterface, \JsonSerializable
      *
      * @return string
```

Comment:
Even if it's practically always a string, shouldn't this be `mixed` too (in all the occurrences in this PR)?

Reply:
I think there are situations where it's justifiable to have a more specific `@return` annotation than the actual type (e.g. when PHP native types don't support the same level of specificity, like `array[]`) but in this instance I went ahead and changed it for the sake of simplicity.

Reply:
I agree that this would be appropriate in most cases, but I'd rather avoid it here since this is based on an external specification.